### DISCUSSION TITLE
Is there a way to know the billing duration of all Lambdas invoked combined?

### DISCUSSION THREAD

**Discussion by dcodesdev** [remotion-dev/remotion#3190] [Q&A] [answered]
Is there a way to know the billing duration of all Lambdas invoked combined?

**Comment by JonnyBurger** [comment, marked as answer]
Implemented for v4.0.74: https://github.com/remotion-dev/remotion/pull/3203

**Reply by dcodesdev** [reply]
This is a very useful feature, thank you so much.
